tests/pom.xml

```diff
@@ -1261,25 +1233,28 @@
                     </log>
                     <wait>
                       <time>${service.startup.timeout}</time>
-                      <log>.*(\[KafkaServer id=1\] started).*</log>
+                      <log>.*(Kafka startTimeMs:).*</log>
                     </wait>
                     <env>
+                      <CLUSTER_ID>q1Sh-9_ISia_zwGINzRvyQ</CLUSTER_ID>
                       <KAFKA_ADVERTISED_LISTENERS>DOCKER_INTERNAL://kafka:9092,DOCKER_EXTERNAL://${docker.host.address}:${kafka.port}</KAFKA_ADVERTISED_LISTENERS>
-                      <KAFKA_BROKER_ID>1</KAFKA_BROKER_ID>
+                      <KAFKA_CONTROLLER_LISTENER_NAMES>CONTROLLER</KAFKA_CONTROLLER_LISTENER_NAMES>
+                      <KAFKA_CONTROLLER_QUORUM_VOTERS>1@127.0.0.1:9093</KAFKA_CONTROLLER_QUORUM_VOTERS>
+                      <KAFKA_NODE_ID>1</KAFKA_NODE_ID>
                       <KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS>0</KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS>
-                      <KAFKA_HEAP_OPTS>-Xms1024m -Xmx1024m</KAFKA_HEAP_OPTS>
                       <KAFKA_INTER_BROKER_LISTENER_NAME>DOCKER_INTERNAL</KAFKA_INTER_BROKER_LISTENER_NAME>
-                      <KAFKA_JVM_PERFORMANCE_OPTS>-server -XX:+UseG1GC -XX:MaxGCPauseMillis=40 -XX:InitiatingHeapOccupancyPercent=35 -XX:+DisableExplicitGC -Djava.awt.headless=true</KAFKA_JVM_PERFORMANCE_OPTS>
-                      <KAFKA_LISTENERS>DOCKER_INTERNAL://0.0.0.0:9092,DOCKER_EXTERNAL://0.0.0.0:9094</KAFKA_LISTENERS>
-                      <KAFKA_LISTENER_SECURITY_PROTOCOL_MAP>DOCKER_INTERNAL:PLAINTEXT,DOCKER_EXTERNAL:PLAINTEXT</KAFKA_LISTENER_SECURITY_PROTOCOL_MAP>
+                      <KAFKA_LISTENERS>DOCKER_INTERNAL://0.0.0.0:9092,CONTROLLER://0.0.0.0:9093,DOCKER_EXTERNAL://0.0.0.0:9094</KAFKA_LISTENERS>
+                      <KAFKA_LISTENER_SECURITY_PROTOCOL_MAP>CONTROLLER:PLAINTEXT,DOCKER_INTERNAL:PLAINTEXT,DOCKER_EXTERNAL:PLAINTEXT</KAFKA_LISTENER_SECURITY_PROTOCOL_MAP>
                       <!-- increase log cleaner check frequency (default is 5min) for test where records shall be removed -->
-                      <KAFKA_LOG_RETENTION_CHECK_INTERVAL_MS>1100</KAFKA_LOG_RETENTION_CHECK_INTERVAL_MS>
+                      <KAFKA_LOG_RETENTION_CHECK_INTERVAL_MS>1000</KAFKA_LOG_RETENTION_CHECK_INTERVAL_MS>
+                      <KAFKA_LOG_SEGMENT_DELETE_DELAY_MS>0</KAFKA_LOG_SEGMENT_DELETE_DELAY_MS>
                       <!-- log level for kafka.server.KafkaServer needs to stay on INFO so that docker-maven-plugin can wait for container startup -->
                       <KAFKA_LOG4J_LOGGERS>kafka.cluster=${hono.kafka.log-level},kafka.controller=${hono.kafka.log-level},kafka.coordinator=${hono.kafka.log-level},kafka.log=${hono.kafka.log-level},kafka.authorizer=${hono.kafka.log-level},kafka.zk=${hono.kafka.log-level},state.change.logger=${hono.kafka.log-level},kafka.server=${hono.kafka.log-level},kafka.server.KafkaServer=INFO</KAFKA_LOG4J_LOGGERS>
```

Comment:
As the Kafka container startup is now detected from a log line like this
```
Kafka[2023-10-28 13:10:16,349] INFO Kafka startTimeMs: 1698498616348 (org.apache.kafka.common.utils.AppInfoParser)
```
and not from a log line of the `kafka.server.KafkaServer` logger anymore, the comment above and the `,kafka.server.KafkaServer=INFO` part should be adapted.

In general, there is now quite a lot of log output from `org.apache.kafka.*` loggers. Also, the `KAFKA_LOG4J_LOGGERS` line above is missing entries for other `kafka.` loggers, e.g. `kafka.network` (I guess this wasn't intended (?)).

Therefore, I think the line could be shortened to this:
```
<KAFKA_LOG4J_LOGGERS>kafka=${hono.kafka.log-level},org.apache.kafka=${hono.kafka.log-level},state.change.logger=${hono.kafka.log-level},org.apache.kafka.common.utils.AppInfoParser=INFO</KAFKA_LOG4J_LOGGERS>
```

Reply:
I am not sure anymore. Maybe I have copied it from an example config for the Kafka 3.5 container image. I will change it to the value you have proposed ...